go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/anchore/syft
 
 go 1.19
 
+replace github.com/microsoft/go-rustaudit => github.com/Microsoft/go-rustaudit v0.0.0-20220730194248-4b17361d90a5
```

Comment:
We definitely don't want to have a replace directive in the `go.mod` -- what is this needed for?

Reply:
This PR has updated the capitalization to be correct, which includes the `go.mod` entries. Why is the `replace` directive needed?

Reply:
agreed, the replace directive is not required if we change the import in the source files that are using this module -- just pushed an update that does this